criu/files-reg.c

```diff
@@ -1403,7 +1403,6 @@ static int get_build_id(const int fd, const struct stat *fd_status,
 	file_header = (Elf_ptr(Ehdr) *) mmap(0, fd_status->st_size,
 						PROT_READ, MAP_PRIVATE, fd, 0);
 	if (file_header == MAP_FAILED) {
-		pr_warn("HERE!\n");
```

Comment:
* we have to log an error in this case
* You added this code in this PR, so you need to merge any fixes of this code in a commit where this code has been introduced. git commit --interactive and git rebase --interactive can help you to do this. We want to have a clean history in the upstream repo.